boards/arm/efr32bg_sltb010a/doc/index.rst

```diff
@@ -0,0 +1,184 @@
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | stimer                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| TRNG      | on-chip    | true random number generator        |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/efr32bg_sltb010a/efr32bg_sltb010a_defconfig``.
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFR32BG SoC has eight gpio controllers (PORTA, PORTB, PORTC, PORTD,
+PORTE and PORTF).
+
+In the following table, the column Name contains Pin names. For example, PE2
+means Pin number 2 on PORTE and #27 represents the location bitfield , as used
+in the board's and microcontroller's datasheets and manuals.
+
++------+-------------+-----------------------------------+
+| Name | Function    | Usage                             |
++======+=============+===================================+
+| PB0  | GPIO        | LED0 (RED)                        |
++------+-------------+-----------------------------------+
+| PB1  | GPIO        | SW0 Push Button PB0               |
++------+-------------+-----------------------------------+
+| PA5  | UART_TX     | UART TX Console VCOM_TX US1_TX #1 |
++------+-------------+-----------------------------------+
+| PA6  | UART_RX     | UART RX Console VCOM_RX US1_RX #1 |
++------+-------------+-----------------------------------+
+
+System Clock
+============
+
+The EFR32BG SoC is configured to use the 38.4 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFR32BG22 SoC has two USARTs.
+USART1 is connected to the board controller and is used for the console.
+
+Programming and Debugging
+*************************
+
+.. note::
+   Before using the kit the first time, you should update the J-Link firmware
+   from `J-Link-Downloads`_
+
+Flashing
+========
+
+The EFR32BG-SLTB010A includes an `J-Link`_ serial and debug adaptor built into the
+board. The adaptor provides:
+
+- A USB connection to the host computer, which exposes a Mass Storage and a
+  USB Serial Port.
+- A Serial Flash device, which implements the USB flash disk file storage.
+- A physical UART connection which is relayed over interface USB Serial port.
+
+Flashing an application to EFR32BG-SLTB010A
+-----------------------------------------
```

Comment:
The length of line 134 does not match the length of line 133, please add `--` to line 134